readthedocs/templates/docsitalia/includes/analytics.html

```diff
@@ -0,0 +1,20 @@
+{% comment %}
+Tacking IDs:
+- UA-122854273-1: Staging - https://docs-italia-staging.teamdigitale.it/
+- UA-122854273-2: Live - https://docs.italia.it/
+{% endcomment %}
+
+{% with "UA-122854273-1" as trackingID %}
```

Comment:
eh questo dobbiamo ricordarci di gestirlo da backend

Reply:
Sì infatti come sempre confidavo in un intervento per sistemare quanto indicato come "missing" nella descrizione della PR.

Reply:
Idealmente sarebbe bello riciclare GLOBAL_ANALYTICS_CODE e DASHBOARD_ANALYTICS_CODE